airflow/contrib/example_dags/example_qubole_operator.py

```diff
@@ -0,0 +1,182 @@
+from airflow import DAG
+from airflow.operators import DummyOperator, PythonOperator, BranchPythonOperator
+from airflow.contrib.operators import QuboleOperator
+from datetime import datetime, timedelta
+import filecmp
+import random
+
+seven_days_ago = datetime.combine(datetime.today() - timedelta(7),
+                                  datetime.min.time())
+
+default_args = {
+    'owner': 'airflow',
+    'depends_on_past': False,
+    'start_date': seven_days_ago,
+    'email': ['airflow@airflow.com'],
+    'email_on_failure': False,
+    'email_on_retry': False
+}
+
+dag = DAG('example_qubole_operator', default_args=default_args)
+
+def compare_result(ds, **kwargs):
```

Comment:
This implementation looks fundamentally incorrect. Either the method should use xcom_pull to fetch command ids of `t1` and `t2` and pass fetched command ids to `get_results` or we should not pass any parameter to this method as it should be able to fetch command_id of it's own task/operation[Mentioned this below as well in `get_log` method of `qubole_hook`]. I would prefer second option.